addon/components/base/bs-nav/link-to.js

```diff
@@ -12,4 +13,15 @@ import ComponentChild from 'ember-bootstrap/mixins/component-child';
  @public
  */
 export default LinkComponent.extend(ComponentChild, {
+  init() {
+    this._super(...arguments);
+    deprecate('The link-to component yielded by bs-nav is deprecated. Use the linkTo property of the yielded item component instead!\n' +
+      'Before: {{#nav.item}}{{#nav.link-to "index"}}foo{{/nav.link-to}}{{/nav.item}}\n' +
+      'After: {{#nav.item linkTo="index"}}foo{{/nav.item}}',
+      false,
+      {
+        id: 'ember-bootstrap.nav.link-to',
+        until: '3.0'
```

Comment:
This will be a very short deprecation period isn't it?

Reply:
Well, who knows? :wink: Don't how quickly all of #780 can be finished...
But 4.0 would be very long I assume, and as this is public API, we can only remove with a major version, right?